src/app/admin-panel/_actions/create-equipment.ts

```diff
@@ -2,28 +2,37 @@
 
 import { fetchPost } from "@/server-endpoints/fetchServer";
 import { createEquipmentSchema } from "@/schemas/equipmentSchemas";
+import { HOST } from "@/server-endpoints/host";
+import { getToken } from "@/auth/getToken";
 
 export async function createEquipment(formData: FormData) {
+  const token = await getToken();
   const name = formData.get("name") as string;
   const id = formData.get("id") as string;
-
   if (id) {
-    console.log("waiting for PUT /equipments/:id to be implemented");
-    // await fetch(`http://localhost:8080/equipments/${id}`, {
-    //   method: "PUT",
-    //   headers: {
-    //     "Content-Type": "application/json",
-    //   },
-    //   body: JSON.stringify({ name }),
-    // });
-    // return;
+    await fetch(`${HOST}/equipments/admin/${id}`, {
```

Comment:
Here you should handle potential errors coming from backend with saving response in a variable and checking response.ok